### ISSUE TITLE
top bar improvements + icon spacing

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-ios#13]
fix icon spacing in compose - should lean towards the right side, with equal spacing between them, leaving enough space for the title (eg `Compose`)

Hide `trash` icon when the message is already in Trash, relevant code (to fix):
```Swift
        if self.path == "[Gmail]/Trash" {
            // self.btnTrash.isHidden = true; // todo - hide the trash btn
        }
```
and to remove:
```Swift
        guard self.path != "[Gmail]/Trash" else { // todo - does not need to be here once trash btn is hidden
            self.showToast("Message is already in trash")
            return
        }
```
Ensure spacing of other icons does not break without the trash icon.

**Comment by tomholub** [comment]
Here broken spacing
![image](https://user-images.githubusercontent.com/6306961/61993327-c780bb00-b09c-11e9-80d6-245c75a67097.png)

Here also broken, should be closer together:
![image](https://user-images.githubusercontent.com/6306961/61993330-d7000400-b09c-11e9-8704-2e9c0cd68ad1.png)

**Comment by RomanKovalchuk2019** [comment]
Fixed in pr #32
